src/main/kotlin/no/nav/syfo/aktivitetskrav/api/AktivitetskravApi.kt

```diff
@@ -74,6 +78,26 @@ fun Route.registerAktivitetskravApi(
 
             call.respond(HttpStatusCode.OK)
         }
+        post("/{$aktivitetskravParam}$forhandsvarselPath") {
```

Comment:
Virket på TH som vi også må lage et endepunkt for forhåndsvarsel uten at det er generert et aktivitetskrav (altså en varient uten `aktivitetskravParam` - ala endepunktet over). Kan jo evt utvide med det i ny PR.

Reply:
Nice! La til en [lapp](https://trello.com/c/o8rxGO5c) på det i boardet og sa ifra til TH 😊